client/src/components/Sidebar/ChatContent.js

```diff
@@ -19,26 +22,23 @@ const useStyles = makeStyles((theme) => ({
     letterSpacing: -0.17,
   },
   notification: {
-    height: 20,
-    width: 20,
     backgroundColor: "#3F92FF",
-    marginRight: 10,
+    marginRight: 20,
     color: "white",
     fontSize: 10,
-    letterSpacing: -0.5,
     fontWeight: "bold",
-    display: "flex",
-    alignItems: "center",
-    justifyContent: "center",
-    borderRadius: 10,
   },
 }));
 
 const ChatContent = (props) => {
   const classes = useStyles();
 
-  const { conversation } = props;
-  const { latestMessageText, otherUser } = conversation;
+  const { conversation, activeConversation } = props;
+  const { latestMessageText, otherUser, unreadMessagesCount } = conversation;
+
+  useEffect(async () => {
+    await props.fetchConversations();
+  }, [latestMessageText]);
```

Comment:
This is probably throwing a warning. Check out this article for [more info](https://dev.to/danialdezfouli/what-s-wrong-with-the-async-function-in-useeffect-4jne).

Reply:
Thanks for sharing